requirements.txt

```diff
@@ -8,3 +11,10 @@ jsonschema==3.0.0a3
 rfc3987
 #required to validate date/time
 #strict-rfc3339
+
+#allows code coverage reporting
+pytest-cov
```

Comment:
@afaulconbridge these packages should not be here but either, manually installed or through setup.py extra requires or requirements-test.txt